### PR TITLE
Update vi-first-steps.md (Issue #2542)

### LINKED ISSUE #2542: In consistency in Step 3 requirement
### Problem
The requirement for info need in Gitter Chat in step 3 is inconsistent with the overview of step 3.

In the overview, it asks for links of github.io and raw.githack links, while in the detail of Step 3, it asks for links of github.io and pull request.

### Steps to reproduce the problem

N/A

### Screenshots

Overview of step 3:
image
Inside step 3:
image

### Proposed solution

Change the overview to the same as inside/detailed step 3. Because the Raw.githack link is available in PR.
The overview will be:
> Make sure that you've linked to your github.io and pull request link on the Gitter chat (https://YourUserName.github.io and `LinkToYourPullRequest`). Also include Raw.githack link in your pull request( https://raw.githack.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md).

### PR DESCRIPTION
**Pull request by lillyxxx**
Fixes #2542 

### Description
Update the requirements for Step 3 in the overview so keep the content consistent.

### Raw.Githack preview link

https://raw.githack.com/lillyxxx/lillyxxx-ole.github.io/update-step3-requirement/#!pages/vi/vi-first-steps.md

### Checklist
- [x] Check for issue number in pull request title
- [x] Are there any unneeded files in the pull request?
- [x] Did they make a branch for their patch?
- [x] Does the pull request actually fix the issue?
- [x] Check the pull request on raw.githack, does it display without any errors?
- [x] Is there any merge conflicts?
- [x] Make sure that people use their GitHub accounts when making commits through git

### CODE REVIEW

**Comment by siqiuma** [comment]
I'm a little confused. Do you mean to change "https://raw.githack.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md" to "https://raw.githack.com/lillyxxx/lillyxxx-ole.github.io/update-step3-requirement/#!pages/vi/vi-first-steps.md"?

**Comment by siqiuma** [comment]
I agree with what is said in this issue. It will be more clear for the reader after the change.

File: pages/vi/vi-first-steps.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Follow the instructions on the [Docker Tutorial page](vi-docker-tutorial.md). **
 
 ## Step 3 - Markdown and Fork Tutorial
 
-Follow the instructions on the [GitHub and Markdown page](vi-github-and-markdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (https://YourUserName.github.io and https://raw.githack.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md).
+Follow the instructions on the [GitHub and Markdown page](vi-github-and-markdown.md). Make sure that you've linked to your github.io and pull request link on the Gitter chat (https://YourUserName.github.io and `LinkToYourPullRequest`). Also include Raw.githack link in your pull request( https://raw.githack.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md).
```
Comment (ChiragAwale) [inline pages/vi/vi-first-steps.md]: Do add the link on the Gitter Chat word itself linking to the Gitter chat page.

Comment (lillyxxx) [reply pages/vi/vi-first-steps.md]: Oops! Sorry about the wrong deletion, I added it now :)

Comment (ChiragAwale) [reply pages/vi/vi-first-steps.md]: No problem. Great work regardless :D

**Pull request merged**